proposals/NNNN-remove-policy-mode-option-from-sdl-core.md

```diff
@@ -0,0 +1,286 @@
+# Remove Policy Mode Option From SDL Core
+
+* Proposal: [SDL-NNNN](NNNN-remove-policy-mode-option-from-sdl-core.md)
+* Author: [Jacob Keeler](https://github.com/jacobkeeler)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / HMI]
+
+## Introduction
+
+Currently the SDL Core project has a build option which allows users to choose between one of three policy modes, `HTTP`, `PROPRIETARY`, and `EXTERNAL_PROPRIETARY`. The main difference between these modes are:
+
+1. The policy update flow is slightly different between each mode
+    - `HTTP` mode does not use the HMI at all in the policy update process
+    - `PROPRIETARY` mode uses the HMI to optionally encrypt the policy table snapshot, while Core handles the PTU retry sequence
+    - `EXTERNAL_PROPRIETARY` mode uses the HMI to optionally encrypt the policy table snapshot and to handle the PTU retry sequence
+2. `EXTERNAL_PROPRIETARY` mode supports a number of extra features, including several policy table fields that aren't implemented in `HTTP` or `PROPRIETARY` mode
+
+The `HTTP` and `PROPRIETARY` policy modes were created for the project first when the project first became open-source as a way of including a version of policies with reduced functionality. It was eventually decided to include the full feature in the open, which was when `EXTERNAL_PROPRIETARY` was added to the project (see [SDL Core 4.3.0](https://github.com/smartdevicelink/sdl_core/releases/tag/4.3.0)).
```

Comment:
```suggestion
The `HTTP` and `PROPRIETARY` policy modes were created for the project when the project first became open-source as a way of including a version of policies with reduced functionality. It was eventually decided to include the full feature in the open, which was when `EXTERNAL_PROPRIETARY` was added to the project (see [SDL Core 4.3.0](https://github.com/smartdevicelink/sdl_core/releases/tag/4.3.0)).
```

Reply:
Adressed in https://github.com/smartdevicelink/sdl_evolution/pull/1170/commits/595aeb015144893eb5748a656cdf28793d49856a